class  B2
{
	int test()
	{
		try
		{
			
		}
		catch(ArithmeticException ex)
		{
			return 20;
		}
		return 10;
	}
}
